README.adoc

```diff
@@ -117,16 +117,32 @@ inside the cluster, you can't access it yourself. If it's available, then it ret
 `UP` status. Similarly, if it's unavailable then it returns a `DOWN` status. When the 
 status is `DOWN`, the microservice is considered to be unhealthy.
 
-The health check for the `system` microservice has already been implemented. It has been 
-set up to become unhealthy for 60 seconds when a specific endpoint is called. This 
-endpoint has been provided for you to observe the results of an unhealthy pod and how 
+[role="code_command hotspot file=1", subs="quotes"] 
+---- 
+#Create `InventoryLivenessCheck` class.#
+`inventory/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java`
+---- 
+
+InventoryLivenessCheck.java
+[source, Java, linenums, role='code_column']
+----
+include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[tags=**;!copyright;]
+----
+
+This liveness check verifies that the heap memory usage is below 90% of the maximum memory.
+If more than 90% of the maxmimum memory is used, a status of `DOWN` will be returned. 
+
+The health checks for the `system` microservice have already been implemented. The `system`
+microservice has been set up to become unhealthy for 60 seconds when a specific endpoint is called. 
+This endpoint has been provided for you to observe the results of an unhealthy pod and how 
 {kube} reacts.
```

Comment:
Change
"The health checks for the `system` microservice have already been implemented. The `system` microservice has been set up to become unhealthy for 60 seconds when a specific endpoint is called. This endpoint has been provided for you to observe the results of an unhealthy pod and how {kube} reacts."
To
"The health checks for the `system` microservice were already implemented. The `system` microservice was set up to become unhealthy for 60 seconds when a specific endpoint is called. This endpoint has been provided for you to observe the results of an unhealthy pod and how {kube} reacts."